### ISSUE TITLE
problem with afFieldValueIs and radio buttons

### ISSUE THREAD

**Issue by wesyah234** [aldeed/meteor-autoform#1051]
When using afFieldValueIs to conditionally show/hide something based on the value of another form control, it fails to work when the other form control is a radio button, but I did get it to work by switching the radio button to a regular drop down select box.  See screencast:
http://screencast-o-matic.com/watch/co1Fr8fqwR
In the screen cast you will see how it works fine to display a message below the select box depending on which value is chosen, but after just switching to the radio button control, it fails to work.  It will sort of work if you edit another control on the form, like I demonstrate by editing the first name field.  It has something to do with events firing, but I don't know enough about this package to solve the issue.  Hopefully someone else can fix!
``````
        {{> afQuickField name='demographics.newold'}}
        {{#if afFieldValueIs name="demographics.newold" value="Previously Seen"}}
        Previously Seen is selected above
        {{else}}
        Previously Seen is NOT selected
        {{/if}}       
`````
schema
``````
  , 'demographics.newold': {
    type: String,
    optional: true,
    label: "New vs. New/Previously Seen",
    autoform: {
      type: 'select',
      options: function () {
        return [
          {label: "New", value: "New"},
          {label: "Previously Seen", value: "Previously Seen"},
        ]
      }
    }
  }
``````

**Comment by TheGoose12** [comment]
Dang I just got the same issue. 

The conditional statement doesn't work with autoform: type: 'select-radio' but does with just type 'select'

**Comment by wesyah234** [comment]
thanks @TheGoose12 for confirming!   Due to lack of activity on this package, I'm pretty close to moving to  
http://viewmodel.meteor.com/
that package has a good amount of support for the showing/hiding of sections based on values of certain form fields.  See:
http://viewmodel.meteor.com/#visibleBk
for example.
Good luck!

**Comment by TheGoose12** [comment]
Hey @wesyah234 
I started a work around based on another related issue I found here:
https://github.com/aldeed/meteor-autoform/issues/861

Not sure if this is a  good long term solution but I'm playing around with it and it seems to work, though it seems hacky... 

Here is my sudo code:

```javascript
## SCHEME :: 
    a: {
        label: "A",
        type: String,
        allowedValues: ['aOption1', 'aOption2'],
        autoform: {
        	 type: "select-radio"
        }
    },    


	b: {
	    label: "B",
	    type: "Boolean-radios",
	    allowedValues: ['no', 'yes'],
		autoform: {
		      type: function() {
		        if(Session.get('form_a_option') == "aOption1"){
				  	console.log("aOption1");
		          	return "hidden";
		          }else{
				     return "";
				  }     
		      }
		    }
			         
	        
	},  
	
	
##EVENT::
if (Meteor.isClient) {
	
	Template.insertMyForm.events({
	  "click #a": function() {
		
		console.log( 'form value: ' + AutoForm.getFieldValue('a', 'insertMyForm') );
		
	    Session.set('form_a_option', AutoForm.getFieldValue('a', 'insertMyForm'));
	    
	    console.log(  "session value: " +Session.get('form_a_option') );

	  }
	});

}	
```

**Comment by TheGoose12** [comment]
Okay getting less clear as I'm having to manually set defaults on the template render 

```javascript
	Template.insertMyForm.rendered = function() {
		
		// Get value on render
		Session.set('form_a_option', AutoForm.getFieldValue('a' , 'insertMyForm'));
	
	}
```

**Comment by aldeed** [comment]
Duplicate of #983, fixed in 5.3.1 release
